api/services/DocumentService.ts

```diff
@@ -0,0 +1,80 @@
+import { ServiceOptions, Service } from 'ts-framework-common';
+import BitcapitalService from './BitcapitalService';
+import { DocumentType } from 'bitcapital-core-sdk';
+import {User} from '../models'
+import UserService from './UserService';
+
+
+export interface DocumentServiceOptions extends ServiceOptions{
+
+}
+
+export default class DocumentService extends Service{
+    protected static instance: DocumentService;
+
+    public async getDocumentStatusForUser(id: string){
+        const bitcapitalClient = BitcapitalService.getInstance({}).bitcapital;
+        const user: User = await UserService.getInstance({}).getUser(id);
+
+        return bitcapitalClient.consumers().findOne(user.bitcapitalId)
+                                .then(consumer => {return consumer.status});
```

Comment:
```suggestion
                                .then(consumer => consumer.status);
```
Arrow functions sem `{}` retornam o valor da instrução por padrão, então você não precisa de um return ali.
Caso você precise retornar um objeto de uma arrow function, você pode fazer assim:
`consumer => ({ abc: consumer.status })`

Reply:
Massa